docs/sources/operator/custom-resource-quickstart.md

```diff
@@ -133,6 +133,10 @@ The full hierarchy of custom resources is as follows:
 
 Deploying a GrafanaAgent resource on its own will not spin up any Agent Pods. Agent Operator will create Agent Pods once MetricsInstance and LogsIntance resources have been created. In the next step, you'll roll out a `MetricsInstance` resource to scrape cAdvisor and Kubelet metrics and ship these to your Prometheus-compatible metrics endpoint.
 
+### Disable feature flags reporting
+
+If you would like to disable the [reporting]({{< relref "../configuration/flags.md/#report-use-of-feature-flags" >}}) reporting usage of feature flags to Grafana, set `disableReporting` field to `true`.
```

Comment:
Double 'reporting' here.

Otherwise LGTM, but would like someone with more insight to verify the many changes in the CRDs.